ZnTrack [0.1.5]

PythonFZ opened the pull request:
- fix #105 
- fix #108 
- fix #99 
- fix #95 
- fix #109

SamTov commented on zntrack/dvc/__init__.py:
```diff
@@ -32,6 +32,7 @@ class params(ZnTrackOption):
 
 class result(ZnTrackOption):
```
class should be documented.

PythonFZ replied:
Added description in 089cf8e

SamTov reviewed (approved):
The changes all look good but there is a class in there that should be documented.

PythonFZ commented:
> The changes all look good but there is a class in there that should be documented.
> 
> In general for the PR can you give a short overview of why you made the changes?

I added the docs to the class you mentioned.
Maybe I updated the description after your review, but this PR fixes the above mentioned issues.
